release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -286,6 +286,75 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure.adoc#persistent-storage-csi-azure-disk[Azure Disk CSI Driver Operator].
+
+[id="ocp-4-10-storage-aws-efs-csi-operator"]
+==== Persistent storage using AWS Elastic File Storage CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic File Storage (EFS). This feature was previously introduced as a Technology Preview feature in {product-title} 4.9 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#persistent-storage-csi-aws-efs[AWS EFS CSI Driver Operator].
+
+[id="ocp-4-10-storage-csi-migration-azure-file"]
+==== Automatic CSI migration supports Microsoft Azure file (Technology Preview)
+
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. This feature now supports automatic migration for the Azure File in-tree plug-in to the Azure File CSI driver.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
+
+[id="ocp-4-10-storage-csi-migration-vsphere"]
+==== Automatic CSI migration supports VMware vSphere (Technology Preview)
+
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. This feature now supports automatic migration for the vSphere in-tree plug-in to the vSphere CSI driver.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
+
+[id="ocp-4-10-storage-fsGroup-pod-timeouts"]
+==== Using fsGroup to reduce pod timeouts
+
+If a storage volume contains many files (~1,000,000 or greater), you may experience pod timeouts.
+
+{product-title} 4.10 introduces the ability to use `fsGroup` and `fsGroupChangePolicy` to skip recursive permission change for the storage volume, thus helping to avoid pod timeout problems.
+
+For more information, see xref:../storage/persistent_storage/understanding-persistent-storage.adoc#storage-persistent-storage-fsGroup[Using fsGroup to reduce pod timeouts].
+
+[id="ocp-4-10-storage-shared-dirs-local-vols"]
+==== Using shared directories with local volumes
```

Comment:
@lpettyjo @gnufied 
Does this worth a new feature here?

Reply:
It always worked. We should probably remove it from release notes.

Reply:
@duanwei33 @gnufied There is part of "fsGroup" and part of "shared local volumes" in this reference. Which one did you want removed?

Reply:
I've deleted the "shared local vols" entry as per our conversation.